Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -1032,6 +1032,29 @@ current polyline feature does not belong to.
 
 \cgalExample{Mesh_3/mesh_polyhedral_domain_with_features_sizing.cpp}
 
+\subsubsection Mesh_WithEdgeDistanceCriterion Approximation Criterion for Feature Edges
+
+The following example shows how to generate a mesh with an approximation error criterion.
+This enables the subdivision of edges that are too far away from the corresponding input sharp features.
+It may result in an output mesh with shorter edges along the input polyline features.
+
+\cgalExample{Mesh_3/mesh_polyhedral_domain_with_features_distance.cpp}
+
+\cgalFigureAnchor{mesh3edge_distance}
+<center>
+<table border=0>
+<tr>
+<td><img border="0" src="./u_arch_no_edge_distance.png" alt="u_arch mesh without edge_distance criterion"></td>
+<td><img border="0" src="./u_arch_with_edge_distance.png" alt="u_arch mesh with edge_distance criterion"></td>
+</tr>
+</table>
+</center>
+\cgalFigureCaptionBegin{mesh3edge_distance}
+Left: the mesh without the `%edge_distance` criterion.
+
+Right: the mesh with the `%edge_distance` criterion set to the same value as `%facet_distance`.
+\cgalFigureCaptionEnd
```

Comment:
Looks like it has been forgotten to add Mesh_3/mesh_polyhedral_domain_with_features_distance.cpp in Mesh_3/examples/Mesh_3/CMakeLists.txt (see warnings in https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-104/logs_master/Mesh_3.log).